components/hab/mac/mac-build.sh

```diff
@@ -30,33 +30,34 @@ install_if_missing() {
     local formula="$pkg"
   fi
 
-  echo $(sudo -u ${SUDO_USER} brew list --versions ${pkg})
+  sudo -u "${SUDO_USER}" brew list --versions "${pkg}"
```

Comment:
There are a number of instances of this: https://github.com/koalaman/shellcheck/wiki/SC2005#useless-echo-instead-of-echo-cmd-just-use-cmd